net/curl/Makefile

```diff
@@ -9,15 +9,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/nls.mk
 
 PKG_NAME:=curl
-PKG_VERSION:=8.6.0
-PKG_RELEASE:=1
+PKG_VERSION:=8.7.1
+PKG_RELEASE:=r1
```

Comment:
why use `r1`?

Reply:
`r` will be appended automatically, do not add it here

Reply:
@1715173329 For multitude of reasons, not a big fan of having different version numbers in the makefile and with the binary, so the build system will need to change to not prepend `r` if it already exists in the revision.

Reply:
oh i see it, thanks for clarification!